src/Resources/Locales/en.json

```diff
@@ -1376,11 +1373,17 @@
                 "edit": "Edit time series",
                 "remove": "Remove time series",
                 "property": "Property",
-                "propertyTypeCastLabel": "Cast property type to number"
+                "propertyTypeCastLabel": "Cast property type to number",
+                "noData": "This time series contains no data to display"
             }
         },
         "viewer": {
-            "chart": "Chart",
+            "chart": {
+                "title": "Chart",
+                "messages": {
+                    "noData": "No numeric historized data to display, try selecting a different time range."
```

Comment:
can we shorten this up a bit? `No data was found for the selected time range`

Reply:
based on content feedback we changed it to "No historized data found in the selected time range." but I kept the numeric part in the translation for chart since it might be misleading if we say "no historized data", in fact it is no numeric data found to display in the chart. fyi @azimamansuri @rinisbet